src/nextjournal/clerk/viewer.cljc

```diff
@@ -575,8 +575,8 @@
                                      {:nextjournal/opts {:id (processed-block-id (str id "-" i "-result"))}}
                                      (-> cell (assoc ::doc doc) (assoc :result x)))))
                     (or (when-some [fgm (-> cell :result :nextjournal/value (get-safe :nextjournal.clerk/fragment))]
-                          (map #(hash-map :nextjournal/value %
-                                          :nextjournal/blob-id (-> cell :result :nextjournal/blob-id)) fgm))
+                          (map-indexed #(hash-map :nextjournal/value %2
+                                                  :nextjournal/blob-id (str (-> cell :result :nextjournal/blob-id) %1)) fgm))
```

Comment:
Not sure faking blob ids is the right move here, but couldn't find a better trick. Tried to provide an unambiguous initial path for the presentation here but didn't lead to any good.